frontends/verilog/verilog_parser.y

```diff
@@ -2569,6 +2574,7 @@ gen_stmt_block:
 		ast_stack.back()->children.push_back(node);
 		ast_stack.push_back(node);
 	} gen_stmt_or_module_body_stmt {
+		SET_AST_NODE_LOC(ast_stack.back(), @1, @1);
```

Comment:
Should this not be:
```suggestion
		SET_AST_NODE_LOC(ast_stack.back(), @2, @2);
```
?  I.e., the location of the `gen_stmt_or_module_body_stmt`?